resources/idlharness.js

```diff
@@ -177,27 +177,9 @@ self.IdlArray = function()
      * might contain a partial interface or includes statement that depends
      * on a later one.  Save these up and handle them right before we run
      * tests.
-     *
-     * .partials is simply an array of objects from WebIDLParser.js'
```

Comment:
Let's keep this comment about `partials` and add one about `includes`

Reply:
OK, I'll rewrite it slightly but keep the gist of it.